index.js

```diff
@@ -1,7 +1,10 @@
-var rp = require("request-promise");
-var minimist = require("minimist");
-var cheerio = require("cheerio");
-var UserAgent = require("user-agents");
+#!/usr/bin/env node
+// w e l c o m e  t o  m y  j a v a s c r i p t  t u t o r i a l
```

Comment:
Horrible joke

Reply:
Sorry but this is a complex and above your IQ joke